### DISCUSSION TITLE
PUE value

### DISCUSSION THREAD

**Discussion by vanessam12** [sb-ai-lab/Eco2AI#3] [Q&A] [answered]
First of all, I would like to thank you for your work! It is an amazing project that I think can contribute to a more sustainable and responsible use of AI. 
To my question: In your work you wrote: "PUE is the optional parameter with default value 1. It is defined manually in eco2AI library." So I would like to use eco2AI in a project I am working on, and I would like to include the PUE value in the calculations. How can I do that? I couldn't find anything in your documentation.

Thank you very much in advance!

**Comment by vladimir-laz** [comment, marked as answer]
Dear  @vanessam12 , thank for feedback on our library. We hope it can help in developing sustainable AI.
By default pue is set to 1.0. In order to set optional pue value you can configure in Tracking initialization:

import eco2ai

tracker = eco2ai.Tracker(
    project_name="YourProjectName", 
    experiment_description="training <your model> model",
    file_name="emission.csv”,
    **pue = <Your pue value>**
    )

Please take into account that this is changing default parameter saved in library settings folder (plese look in readme -> use examples) and it will be used by default for next experiments until changed explicitly.

Further documentation you can get via help(eco2ai.Tracker) function or freely ask us in discussion.
Good luck in your further experiments!
